src/readTextFileAsync.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs').promises;
+
+async function readTextFileAsync(path) {
+  try {
+    return await fs.readFile(path, 'utf8');
```

Comment:
Good. It's very important to add `await` here. Without it, the `catch` block would catch the errors from `fs.readFile`. :+1: